### PR TITLE
Add pytorch py

### PR DESCRIPTION
**Pull request by kobrineli**

### CODE REVIEW

File: projects/pytorch-py/build_text.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+set -e
+
+cd /text
+
+ASAN_LIB=$(find /usr/local/lib -name 'asan_with_fuzzer.so')
+
+# Clean previous build
+ASAN_OPTIONS=detect_leaks=0 LD_PRELOAD=$ASAN_LIB \
+    CC=$CC CXX=$CXX CFLAGS=$CFLAGS CXXFLAGS=$CXXFLAGS \
```
Comment (kobrineli) [inline projects/pytorch-py/build_text.sh:26]: maybe remove these variables

File: projects/pytorch-py/build_vision.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+set -e
+
+cd /vision
+
+ASAN_LIB=$(find /usr/local/lib -name 'asan_with_fuzzer.so')
+
+# Clean previous build
+ASAN_OPTIONS=detect_leaks=0 LD_PRELOAD=$ASAN_LIB \
+    CC=$CC CXX=$CXX CFLAGS=$CFLAGS CXXFLAGS=$CXXFLAGS \
```
Comment (kobrineli) [inline projects/pytorch-py/build_vision.sh:26]: maybe remove these variables.

File: projects/pytorch-py/build_text.sh (hunk 1 above)
Comment (apach301) [reply projects/pytorch-py/build_text.sh:26]: remove clean stage

File: projects/pytorch-py/build_vision.sh (hunk 2 above)
Comment (apach301) [reply projects/pytorch-py/build_vision.sh:26]: remove clean stage

File: projects/pytorch-py/build_text.sh (hunk 1 above)
Comment (kobrineli) [reply projects/pytorch-py/build_text.sh]: done

File: projects/pytorch-py/build_vision.sh (hunk 2 above)
Comment (kobrineli) [reply projects/pytorch-py/build_vision.sh]: done

File: projects/pytorch-py/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+FROM sydr/ubuntu20.04-sydr-fuzz
+
+# Install build dependencies.
+RUN pip3 install astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions
+
+# Clone target from GitHub.
+RUN git clone https://github.com/pytorch/pytorch/ /pytorch
+RUN cd /pytorch && \
+    git checkout 22e2fd554cf370765d4c44fe2b99c8bb6e42b0bb && \
```
Comment (apach301) [inline projects/pytorch-py/Dockerfile:25]: Select PyTorch version to be compatible with vision and text

Comment (kobrineli) [reply projects/pytorch-py/Dockerfile]: torch text still fails, so returned `|| true` back

**Pull request merged**